src/commands/publish.ts

```diff
@@ -505,13 +497,20 @@ export async function publishMain(argv: PublishOptions): Promise<any> {
       return undefined;
     }
 
-    await printRevisionSummary(
-      zeus,
-      revision,
-      targetConfigList.map(t => t.name || '__undefined__')
-    );
+    // TODO: Handle new Artifacts stores
+    // We only ask Zeus now for artifacts
+    if (statusProviderName === StatusProviderName.Zeus) {
+      await printRevisionSummary(zeus, revision);
+      await checkRequiredArtifacts(zeus, revision, config.requireNames);
```

Comment:
Just a note: we can do this for now, but in the future we want to allow arbitrary statusProvider/artifactProvider combinations.